addons/website/static/src/snippets/s_facebook_page/000.js

```diff
@@ -17,10 +17,10 @@ const FacebookPageWidget = publicWidget.Widget.extend({
         this.options.wysiwyg && this.options.wysiwyg.odooEditor.observerUnactive();
 
         var params = _.pick(this.$el.data(), 'href', 'height', 'tabs', 'small_header', 'hide_cover', 'show_facepile');
-        if (!params.href) {
-            return def;
-        }
         params.width = utils.confine(Math.floor(this.$el.width()), 180, 500);
+        params.height = params.height || 70;
+        params.hide_cover = params.hide_cover || true;
+        params.href = params.href || 'https://www.facebook.com/Odoo';
```

Comment:
It was working because the facebook option, which has a hardcoded `data-page-url="https://www.facebook.com/Odoo", transfers its data attribute to the snippet and refreshes the public widget when started

Reply:
So the code that transfers that should have been removed I guess... Anyways I restored the code as before as not needed since we won't be starting animation before the options onBuilt & stuff.